shopelectro/tests/tests_wholesale.py

```diff
@@ -2,37 +2,31 @@
 
 from django.test import TestCase
 
-from shopelectro.models import Product, Category
+from shopelectro.models import Product
 from shopelectro.cart import WholesaleCart
 
 
 class WholesaleCartTest(TestCase):
-    @classmethod
-    def setUpTestData(cls):
-        """Create products and category for them."""
-        category = Category.objects.create(name='Test')
-        cls.item_quantity = 2
-        cls.products = cls.first_product, cls.second_product = [
-            Product.objects.create(
-                name='Product {}'.format(i),
-                category=category,
-                price=1550.00,
-                wholesale_small=1145.74,
-                wholesale_medium=1082.59,
-                wholesale_large=992.38
-            ) for i in range(cls.item_quantity)
-            ]
+
+    fixtures = ['dump.json']
+
+    def __init__(self, *args, **kwargs):
+        self.item_quantity = 2
+
+        super(WholesaleCartTest, self).__init__(*args, **kwargs)
 
     def setUp(self):
         """Get session for test."""
+        self.products = Product.objects.all()[:self.item_quantity]
+        self.first_product, self.second_product = self.products
         self.session = self.client.session
 
     @property
     def cart(self):
         """Return Cart object for test."""
         return WholesaleCart(self.session)
 
-    def wholesale_quantity(self, price_type):
+    def get_wholesale_quantity(self, price_type, product):
         """
         Return the approximate number of products with wholesale price type for
         tests.
```

Comment:
Return minimal products count, that we should put in cart to get assigned price_type